src/main/assembly/dist/install/db-tables.sql

```diff
@@ -1,8 +1,11 @@
+CREATE TYPE Origin AS ENUM ('SWORD2', 'API', 'SMD');
```

Comment:
This is not gonna work; see EASY-2300.

My current proposal for that bug (which is similar to the one you introduced here) can be found at https://github.com/DANS-KNAW/easy-deposit-properties/pull/5